rmw_connextdds_common/include/rmw_connextdds/dds_api.hpp

```diff
@@ -46,9 +46,20 @@ enum RMW_Connext_MessageType
   RMW_CONNEXT_MESSAGE_REPLY
 };
 
+struct RMW_Connext_WriteParams
+{
+  DDS_Time_t timestamp{DDS_TIME_INVALID};
+  int64_t sequence_number{0};
+};
+
 RMW_CONNEXTDDS_PUBLIC extern const char * const RMW_CONNEXTDDS_ID;
 extern const char * const RMW_CONNEXTDDS_SERIALIZATION_FORMAT;
 
+rmw_ret_t
+rmw_connextdds_get_current_time(
+  DDS_DomainParticipant * domain_participant,
+  struct DDS_Time_t * current_time)
```

Comment:
@cwecht There's a missing `;` here which [causes the build to fail](https://ci.ros2.org/job/ci_linux/19984/console#console-section-10).

Reply:
Fixed.